### DISCUSSION TITLE
ARC best practices

### DISCUSSION THREAD

**Discussion by eyarz** [actions/actions-runner-controller#1751] [Questions]
Is there a list of known best practices for configuring the different ARC objects (e.g. `RunnerDeployments`, `RunnerSet`, etc.)?
I see there are some tips in the readme, but not something official about recommended configuration setup.
I'm asking because I want to codify the best practices, and then I will be able to prevent misconfiguration locally or during CI/CD.

**Comment by jurgen-weber-deltatre** [comment]
I only just found this today, the #1 question I am trying to answer is.. What is the difference between RunnerDeployments and RunnerSets (beyond a k8s deployment/k8s statefulset) and when would I want to use either and in what situations?

and if the difference between Deployments and Sets is not deployments and statefulsets, well.. You know what they say about assumptions. :)
